notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-095_settling_big_vibrations.ipynb

```diff
@@ -0,0 +1,664 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #67.        rolling = 50
</pre><p>Did you check how much the settle time determination depend on this parameter ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
It does indeed depend. Actually, 10 is a better determination to try to get as close as possible to the slew end (otherwise, 50 was almost 1 second which is already the requirement).